### PR TITLE
Add a docstring for StatMesh

### PR DESCRIPTION
**Pull request by gonuke**

### CODE REVIEW

File: pyne/mesh.py [hunk 1]
```diff
@@ -1433,6 +1433,20 @@ def tag_cell_fracs(self, cell_fracs):
...
+    mathematical operations that are performed on multiple meshes.
+
+    A StatMesh assumes that each value being operated upon also has a
+    statistical error associaed with it, and forces operations on the
+    statistical error as well.
+
+    For example, when to quantities are added together, c = a + b, the
+    statistical error of c is found by combining the statistical errors of a
+    and b.
+
```
Comment (pshriwise) [inline pyne/mesh.py:1447]: We may want a note in here about how it finds error values associated with data (`tag name` + "`_rel_error`").

**Review by pshriwise — COMMENTED** [review]
One optional change. Otherwise LGTM.

File: pyne/mesh.py (hunk 1 above)
Comment (gonuke) [reply pyne/mesh.py:1447]: done

**Review by pshriwise — APPROVED** [review]
The doc string looks good. Thanks @gonuke!

**Pull request merged**